### PR TITLE
Added keyboard field navigation for username, password textfield in bookmark view

### PR DESCRIPTION
**Pull request by hosy**
## Description
Added a `inputAccessoryView ` for username and password text field for field navigation between these rows, also setting a `ReturnTypeKey`.

## Related Issue
#48 

## Motivation and Context
Better navigation between text fields form keyboard, because next web view for oAuth 2 does also have this field navigators.

## How Has This Been Tested?
- add or edit a bookmark
- set focus on username or password text field

## Screenshots (if appropriate):
Simulator Screen Shot - iPhone 8 - 2019-03-10 at 11 31 35

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING**](https://github.com/owncloud/ios-app/blob/master/CONTRIBUTING.md) document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: ownCloud/Bookmarks/BookmarkViewController.swift [hunk 1]
```diff
@@ -140,6 +155,7 @@ class BookmarkViewController: StaticTableViewController {
 		// Credentials section + rows
 		usernameRow = StaticTableViewRow(textFieldWithAction: { [weak self] (_, sender) in
 			if (sender as? UITextField) != nil, self?.bookmark?.authenticationData != nil {
+				//sender.inputAccessoryView = inputToolbar
```
Comment (mneuwert) [inline ownCloud/Bookmarks/BookmarkViewController.swift]: @hosy remove dead code

File: ownCloud/Bookmarks/BookmarkViewController.swift [hunk 2]
```diff
@@ -727,6 +743,21 @@ class BookmarkViewController: StaticTableViewController {
 	func isAuthenticationMethodTokenBased(_ authenticationMethodIdentifier: OCAuthenticationMethodIdentifier) -> Bool {
 		return authenticationMethodTypeForIdentifier(authenticationMethodIdentifier) == OCAuthenticationMethodType.token
 	}
+	
+	// MARK: - Keyboard AccessoryView
+	@objc func toogleTextField (_ sender: UIBarButtonItem) {
```
Comment (mneuwert) [inline ownCloud/Bookmarks/BookmarkViewController.swift]: @hosy Instead of comparing `sender.style` I would suggest to create separate action methods

Comment (hosy) [reply ownCloud/Bookmarks/BookmarkViewController.swift]: Make sense in this case

**Comment by jesmrec** [comment]
For me it is OK, but a tiny detail:

In the "Add account" scenario, you are lighting/shadowing the arrows like:

Screen Shot 2019-03-21 at 13 51 16

Screen Shot 2019-03-21 at 13 51 23

In case of "Edit account" with a basic auth server, there is no choice to move up/down through text fields, because only `Password` is enabled:

Screen Shot 2019-03-21 at 13 52 56

so, the blue up arrow does not do nothing. It should be shadowed/hidden...

Apart of that, this is OK for me.

**Comment by hosy** [comment]
@jesmrec I just fixed it. Accessory View is now only available, if the needed text fields are enabled.

**Comment by jesmrec** [comment]
Approved

**Pull request merged**